### PR TITLE
Add a new script to parse image shas for new tags.

### PR DESCRIPTION
**Pull request by prameshj**
This can be used to generate the diff for promoting images in https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io/images/k8s-staging-dns

### CODE REVIEW

File: parse-image-sha.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# This script pulls the docker images for dns containers and outputs the sha256
```
Comment (MrHohn) [inline parse-image-sha.py:1]: Should we prepend the Apache license boilerplate here?

**Review by MrHohn — APPROVED** [review]
Thanks for adding this - I'm looking forward to mirror this in other repos as well :)

File: parse-image-sha.py (hunk 1 above)
Comment (prameshj) [reply parse-image-sha.py:17]: Good idea, done.

**Review by MrHohn — APPROVED** [review]
/lgtm

**Pull request merged**